audio/midiparser.h

```diff
@@ -377,7 +381,14 @@ class MidiParser {
 		  * Any active notes registered by this parser will still be turned
 		  * off.
 		  */
-		 mpDisableAllNotesOffMidiEvents = 6
+		 mpDisableAllNotesOffMidiEvents = 6,
+
+		 /**
+		  * Does not automatically start playback after parsing MIDI data
+		  * or setting the track. Use startPlaying to start playback.
+		  * Note that not every parser implementation might support this.
+		  */
+		  mpDoNotAutoStartPlayback = 7
```

Comment:
How would you feel about mpDisableAutoStartPlayback name for consistency? Too much of a nitpick?

Reply:
Yes, that's a better name.